pkg/streams/streamsGet.go

```diff
@@ -18,24 +20,40 @@ import (
 	"github.com/galasa-dev/cli/pkg/streamsformatter"
 )
 
-func GetStreams(streamName string, apiClient *galasaapi.APIClient, console spi.Console) error {
+var (
+	formatters = createFormatters()
+)
+
+func GetStreams(streamName string, format string, apiClient *galasaapi.APIClient, console spi.Console) error {
+
+	var err error
+	var streamData []galasaapi.Stream
+
+	if streamName != "" {
+		streamName, err = validateStreamName(streamName)
+	}
 
-	streamData, err := getStreamsFromRestApi(streamName, apiClient)
 	if err == nil {
-		err = formatFetchedStreamsAndWriteToConsole(streamData, console)
+		streamData, err = getStreamsFromRestApi(streamName, apiClient)
+		if err == nil {
+			err = formatFetchedStreamsAndWriteToConsole(streamData, console, format)
+		}
 	}
 
 	return err
 
 }
 
-func formatFetchedStreamsAndWriteToConsole(streams []galasaapi.Stream, console spi.Console) error {
+func formatFetchedStreamsAndWriteToConsole(streams []galasaapi.Stream, console spi.Console, outputFormatString string) error {
 
-	summaryFormatter := streamsformatter.NewStreamsSummaryFormatter()
-	outputText, err := summaryFormatter.FormatStreams(streams)
+	var formattedOuptut string
+	chosenFormatter, err := validateFormatFlag(outputFormatString)
```

Comment:
I think it would be better to validate the format flag at the very start to avoid having to send a GET request to the API server only to fail because the user provided an invalid format value